### ISSUE TITLE
RDP Client tests failed with same error

### ISSUE THREAD

**Issue by ligurio** [microsoft/WindowsProtocolTestSuites#215]
```
Initialization method Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrTestSuite.BaseTestInitialize threw exception. System.IO.FileNotFoundException: System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.SpecExplorer.Runtime, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified..
```

Looks like it is requires SpecExplorer, but according to [prerequisites](https://github.com/Microsoft/WindowsProtocolTestSuites#prerequisites) SpecExplorer is not required for RDP tests.

Windows Protocol Test Framework 1.0.7500.0
Microsoft RDP Client Endpointe Test Suite 3.20.1.0
Protocol Test Manager 3.20.1.0

**Comment by huiren-jiang** [comment]
Hi Sergey,
 
Thanks for reporting this issue.
 
Just as you observed, Spec Explorer is now required by all test suites even if they do not include model-based tests. We will update the corresponding prerequisites of PTF and test suites later.
 
Best Regards,
 
Windows Protocol Test Suites Development Team

**Comment by ligurio** [comment]
Updated versions and problem has gone. Thanks!

**Comment by huiren-jiang** [comment]
Hi Sergey,
 
Thanks for your contribution. The update to prerequisites is available on staging by commit 514af74.
 
Best Regards,
 
Windows Protocol Test Suites Development Team

**Comment by yazeng** [comment]
Hi Sergey,
Since this thread has been inactive for a long time, we close it for now. Feel free to reopen it if the problem remains.
Windows Protocol Test Suites Development Team
